### PR TITLE
[FIX] date_range : change license from AGPL-3 to LGPL-3, because l10n-france/l10n_fr_fec_oca/12.0 (LGPL) depends on date_range

### PR DESCRIPTION
**Pull request by legalsylvain**
Trivial Change.
for the time being : 
- [l10n_fr_fec_oca](https://github.com/OCA/l10n-france/tree/12.0/l10n_fr_fec_oca) is LGPL-3 and depends on date_range (AGPL-3)
- we don't have the copyright of l10n_fr_fec_oca to change the licence.

**3 options :** 
- remove ``l10n_fr_fec_oca`` from OCA/l10n-france. that solution looks quite radical
- remove dependency to date_range. That solution looks not desirable
- change the license of date_range. (solution simple)

Thanks for your quick review.

(This PR allow to use new oca CI on l10n-france repo. https://github.com/OCA/l10n-france/pull/446)

CC : @sbidoul, @alexis-via

### CODE REVIEW

**Review by sbidoul — APPROVED** [review]

**Comment by legalsylvain** [comment]
hi. There is some trouble in the base_duplicate_security_group. 
See log : 
https://github.com/OCA/server-ux/actions/runs/4893071347/jobs/8735587206?pr=654#step:8:364

```
 2023-05-05 11:56:41,624 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ` Whether or not the duplicate button is available The button should be accessible depending on the user permissions. 
2023-05-05 11:56:41,730 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Chrome pid: 921 
2023-05-05 11:56:41,732 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Issuing json command version 
2023-05-05 11:56:41,732 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Url : http://127.0.0.1:52067/json/version 
2023-05-05 11:56:42,000 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Browser version: HeadlessChrome/112.0.5615.121 
2023-05-05 11:56:42,000 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Issuing json command  
2023-05-05 11:56:42,000 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Url : http://127.0.0.1:52067/json 
2023-05-05 11:56:42,004 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Chrome headless temporary user profile dir: /tmp/tmpth_r_o48_chrome_odoo 
2023-05-05 11:56:42,004 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Websocket url found: ws://127.0.0.1:52067/devtools/page/86AB1A7FC4765CF2F04CBA491CA9126E 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ERROR 
2023-05-05 11:56:42,283 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ====================================================================== 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ERROR: test_duplicate_button (odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup) 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ` Whether or not the duplicate button is available The button should be accessible depending on the user permissions. 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: Traceback (most recent call last): 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/__w/server-ux/server-ux/base_duplicate_security_group/tests/test_base_duplicate_security_group.py", line 21, in test_duplicate_button 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     login="admin", 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 1083, in browser_js 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self.start_browser(self._logger) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 989, in start_browser 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     cls.browser = ChromeBrowser(logger) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 603, in __init__ 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self._open_websocket() 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 756, in _open_websocket 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self.ws = websocket.create_connection(self.ws_url) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_core.py", line 606, in create_connection 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     websock.connect(url, **options) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_core.py", line 253, in connect 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self.handshake_response = handshake(self.sock, url, *addrs, **options) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_handshake.py", line 57, in handshake 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     status, resp = _get_resp_headers(sock) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_handshake.py", line 148, in _get_resp_headers 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     raise WebSocketBadStatusException("Handshake status %d %s", status, status_message, resp_headers) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ` websocket._exceptions.WebSocketBadStatusException: Handshake status 403 Forbidden 
2023-05-05 11:56:42,285 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: Ran 1 test in 0.660s 
2023-05-05 11:56:42,285 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: FAILED 
2023-05-05 11:56:42,285 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group:  (errors=1) 
2023-05-05 11:56:42,285 890 ERROR odoo odoo.modules.module: Module base_duplicate_security_group: 0 failures, 1 errors 
2023-05-05 11:56:42,286 890 INFO odoo odoo.modules.module: odoo.addons.base_import_security_group.tests.test_base_import_security_group running tests. 

````
Sometimes it fails, some times it success. 
Two options : 
-  fix the test in  ``base_duplicate_security_group``
- disable the test in ``base_duplicate_security_group``.

@chienandalu, as the maintainer of this module, an idea ? 

thanks !

**Comment by chienandalu** [comment]
Looks like a random error. Can you relaunch that job?

**Comment by legalsylvain** [comment]
Try 1 : Odoo failed / OCB failed
Try 2 : Odoo failed / OCB succeeded
Try 3 : Odoo failed / OCB succeeded

What do you think @chienandalu ?

**Comment by chienandalu** [comment]
To be honest I have no clue :confused: As we don't maintain this version anymore, you can put it to ignore in tests

File: base_duplicate_security_group/tests/test_base_duplicate_security_group.py
Comment (chienandalu) [inline base_duplicate_security_group/tests/test_base_duplicate_security_group.py]: Isn't there an ignore directive for the Github actions?

Comment (chienandalu) [reply base_duplicate_security_group/tests/test_base_duplicate_security_group.py]: Some like in here: https://github.com/OCA/sale-workflow/blob/0ee4c7eb7fd712b744a2cb9b47329d7dacf7bd9f/.github/workflows/test.yml#L46

Comment (legalsylvain) [reply base_duplicate_security_group/tests/test_base_duplicate_security_group.py]: Thanks for your reactivity.
Done. And CI is green. could you approve, if it's OK for you ? 

regards.

**Review by chienandalu — APPROVED** [review]
:smile: :+1:

**Comment by legalsylvain** [comment]
@sbidoul : could you merge, to unblock CI usage on l10n-france ? 
thanks !

**Comment by sbidoul** [comment]
@legalsylvain let's give it a few days so other `date_range` contributors have a chance to chime in if they have concerns with this licence change?

**Comment by legalsylvain** [comment]
> @legalsylvain let's give it a few days so other date_range contributors have a chance to chime in if they have concerns with this licence change?

You're totally right. CC : @lmignon, current maintainer.

Note : that change should be done on 10 / 12 / 14 to fix the license issue.

image

**Review by lmignon — APPROVED** [review]

**Comment by legalsylvain** [comment]
thanks for your review ! 

/ocabot merge patch

**Comment by legalsylvain** [comment]
@lmignon, @sbidoul : it's up to you here.

**Pull request merged**
